todoapp/app/build.gradle

```diff
@@ -68,52 +67,51 @@ android {
  all versions in a single place. This improves readability and helps managing project complexity.
  */
 dependencies {
-    // App's dependencies, including test
-    compile "com.android.support:appcompat-v7:$rootProject.supportLibraryVersion"
-    compile "com.android.support:cardview-v7:$rootProject.supportLibraryVersion"
-    compile "com.android.support:design:$rootProject.supportLibraryVersion"
-    compile "com.android.support:recyclerview-v7:$rootProject.supportLibraryVersion"
-    compile "com.android.support:support-v4:$rootProject.supportLibraryVersion"
-    compile "com.google.guava:guava:$rootProject.guavaVersion"
+    // App dependencies
+    implementation "androidx.appcompat:appcompat:$appCompatVersion"
```

Comment:
Nit: Sort alphabetically

Reply:
That's a good idea for the root branch, but this one would produce diffs. I'll TODO it.